tests/snippets/attr.py

```diff
@@ -10,6 +10,15 @@ class A:
 assert hasattr(a, 'b')
 assert a.b == 10
 
+a.c = 50
+assert hasattr(a, 'c')
+assert a.c == 50
+
+# test delete attribute with del keyword
+del a.c
+with assert_raises(AttributeError):
+    _ = a.c
+
```

Comment:
Could you add the snippet the from the explanation in the pull request as well? Class attributes probably take another code path as instance attributes.

I mean this snippet:
```python
class C:
    x = 10

...
```

Reply:
Thank you, I added it.